docs/toc.yml

```diff
@@ -136,25 +134,5 @@ entries:
   - file: main/src/par/README
     title: Partition Manager
 
-- file: user/GettingStarted
-  title: Getting Started with OpenROAD Flow
-  entries:
-  - file: user/BuildWithDocker
-    title: Build With Docker
-  - file: user/BuildLocally
-    title: Build Locally
-  - file: user/UserGuide
-    title: User Guide
-  - file: user/FlowVariables
-    title: Flow Variables
-  - file: contrib/Metrics
-    title: Metrics
-  - file: user/InstructionsForAutoTuner
-    title: AutoTuner
-
-
 - file: user/FAQS
   title: FAQs
-
-- file: tutorials/FlowTutorial
-  title: Flow Tutorial
```

Comment:
Let's replace this with a new file called `tutorials/index.md`. In this new file, we should reference the ORFS dos tutorials and later we would be able to add OR-only tutorials.